src/test/ui/borrowck/borrowck-for-loop-uninitialized-binding.stderr

```diff
@@ -1,3 +1,11 @@
+warning: empty for iterator
```

Comment:
Might be better to `#[allow]` the lint in `ui` tests as well, so that when lint output changes unrelated tests don't.